### PR TITLE
Added local web server for Tracer log viewer

### PR DESCRIPTION
**Pull request by richardkasongo2003**
#### PR description:
- Introduced a python based web server that users can use to avoid the web browsers from needing to access the files locally

#### PR validation:

-ran the server locally on a windows machine

### CODE REVIEW

File: FWCore/Services/web/serv.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            self.wfile.write(bytes("File not found", 'utf-8'))
+        
+        except Exception as e:
+            self.send_response(500)
+            self.send_header("Content-type", "text/html")
+            self.end_headers()
+            self.wfile.write(bytes(f"Internal server error: {str(e)}", 'utf-8'))
+
+httpd = HTTPServer(('localhost', 65432), Serv)
+print("Server started at localhost:65432")
```
Comment (Dr15Jones) [inline FWCore/Services/web/serv.py:38]:
```suggestion
print("Server started at http://localhost:65432")
```

That way a person could just cut-n-paste this directly into their web-browser

File: FWCore/Services/web/serv.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
+from http.server import HTTPServer, BaseHTTPRequestHandler
```
Comment (Dr15Jones) [inline FWCore/Services/web/serv.py:1]: Please rename the file to `server.py` as that is clearer and also matches the documentation.

**Comment by Dr15Jones** [comment]
please test

**Comment by Dr15Jones** [comment]
+1

**Comment by rappoccio** [comment]
+1

**Pull request merged**